### PR TITLE
Merged Inverter DBC File to App_CanMsgs.dbc for Single Inverter

### PR DESCRIPTION
**Pull request by timbitss**
### Summary
- Merged PM100DZ Inverter DBC File "[20210505 RMS PM and RM CAN DB.dbc](https://app.box.com/s/gu8b4utzjsrx17in8vipgfqxpt8fuf1u/folder/134049162381)"  to App_CanMsgs.dbc for Single Inverter. 

### Changelist 
- Modified App_CanMsgs.dbc to include inverter node (INV1) and messages. Added additional comments for clarity.
- Modified DCM's main.c file to be compatible with the new auto-generated CAN code. 
- Modified test state machine for compatibility with non-period inverter message.

### Testing Done
- Built DCM.elf with no errors.
- Ran gtest with passing tests.
 
### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [x] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### Notes

- Inverter Messages that were **not** merged To App_CanMsgs.dbc:

1. BO_ 471 M187_U2C_Command_Txd: 8 INV
2. BO_ 469 M188_U2C_Message_Rxd: 8 U2C
3. BO_ 514 BMS_Current_Limit: 8 BMS

### CODE REVIEW

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (timbitss) [reply scripts/codegen/CAN/App_CanMsgs.dbc:346]: No, it should not because we've set the Torque_Request signal as a floating-point value

`SIG_VALTYPE_ 206 Torque_Request : 1;`

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (alxshev) [inline scripts/codegen/CAN/App_CanMsgs.dbc:433]: @Marko/Prom do these 2 lines do anything?

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (alxshev) [inline scripts/codegen/CAN/App_CanMsgs.dbc:160]: I think that some of these signals don't need to be here, things like:
1. INV1_Temperature_Set
2. INV1_Flux_ID_IQ_Info
3. INV1_Internal_Voltages
4. INV1_Internal_Voltages

But they may be useful for testing--I'm not sure inverter information the DCM would be interested in. @markomarkusi  thoughts?

**Review by alxshev — APPROVED** [review]
Looks good. Marko or Prom should review this before merging as well, left some high-level comments I had

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (pasitlaothamatas) [reply scripts/codegen/CAN/App_CanMsgs.dbc:160]: Its unclear to me from the specs which CAN messages need to be read by the DCM. If most of these signals are for debugging we should probably specify them to be DEBUG messages.

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (pasitlaothamatas) [reply scripts/codegen/CAN/App_CanMsgs.dbc:433]: This may be checking for overflow in can tx/rx mailboxes. Were these accidentally removed?

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (pasitlaothamatas) [inline scripts/codegen/CAN/App_CanMsgs.dbc:671]: How did we decide on these CAN msg frequencies?

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (timbitss) [reply scripts/codegen/CAN/App_CanMsgs.dbc:433]: There is no _BO_ 2_ message ID in the original file so I don't think they were accidentally removed.

File: scripts/codegen/CAN/App_CanMsgs.dbc
Comment (timbitss) [reply scripts/codegen/CAN/App_CanMsgs.dbc:671]: Those message cycle times were set by the inverter manufacturer. I don't think you can change them because it is the inverter that is broadcasting these messages. 

We have not decided yet on the actual torque request msg frequency but it is currently set at 5 ms by default.

**Pull request closed without merging**
